http.go

```diff
@@ -17,11 +17,12 @@ type resultContainer struct {
 	Success  bool      `json:"success"`
 	Message  string    `json:"message"`
 	Comments []Comment `json:"comments,omitempty"`
+	Count    *int       `json:"count,omitempty"`
```

Comment:
Int is simple data type so it should be a value.

You can leave out the 'omitempty' as well. Count will default to 0, so if there are no comments then the default value is reasonable.